src/main/groovy/edu/oregonstate/mist/inventory/db/InventoryDAOWrapper.groovy

```diff
@@ -56,14 +62,21 @@ class InventoryDAOWrapper {
      * @param selfLinkBase
      * @return
      */
-    private ResourceObject inventoryBuilder(Inventory inventory, URI selfLinkBase) {
-        inventory.apiQueryParams = inventoryDAO.getFields(QUERY_DB_TYPE, inventory.id)
+    private ResourceObject inventoryBuilder(Inventory inventory) {
+        inventory.apiQueryParams = inventoryDAO.getFields(
+                QUERY_DB_TYPE,
+                inventory.id,
```

Comment:
Not a typo. Since fields (whether provided or being used in a query) are stored in the same table, the field record in the DB has a parent ID and the related inventory ID. For api query params, the parentID is also the inventoryID. I updated the method comment in the DAO interface to make this more clear.